server/integrationtests/pluginteststore.go

```diff
@@ -269,14 +269,15 @@ func (s *PluginTestStore) SearchUserChannels(teamID, userID, query string) ([]*m
 }
 
 func (s *PluginTestStore) GetChannel(teamID, channel string) (*mmModel.Channel, error) {
-	if channel == "valid-channel-id" {
+	switch channel {
```

Comment:
Seems a bit excessive to use switch even for a n if-else-if with just two conditions. If its some linter rule and not required for FIPS compliance, we can probably alter it outside of this PR as it doesn't block this change,